### PR TITLE
Update SENSOR.yaml

### PR DESCRIPTION
**Pull request by RathishKJ**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml [hunk 1]
```diff
@@ -186,3 +186,29 @@ SENSOR_ZA_PML:
...
   - zone_air_voc_concentration_sensor
+  
+SENSOR_COC:
+  description: "Carbon monoxide monitoring and control"
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - COC
+  uses:
+  - zone_air_co_concentration_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml:197]: you implemented COC, this isnt necessary.

File: ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml [hunk 2]
```diff
@@ -186,3 +186,29 @@ SENSOR_ZA_PML:
...
+ 
+SENSOR_OCO2M:
+  description: "Outside Carbon dioxide monitoring and control"
+  is_canonical: true
+  implements:
+  - /SENSOR
+  - OCO2M
+  uses:
+  - outside_air_co2_concentration_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml:206]: you defined this in OCO2M, this isnt needed.

**Review by tasodorff — CHANGES_REQUESTED** [review]
Remove fields from type definitions for those wehre you implment an abstract type with those exact points.

File: ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml (hunk 1 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml:197]: Removed implemented COC.

File: ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml (hunk 2 above)
Comment (RathishKJ) [reply ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml:206]: removed OCO2M

**Review by tasodorff — APPROVED** [review]

**Comment by tasodorff** [comment]
@RathishKJ resolve conflicts for merger

**Pull request merged**